src/modules/web3/model/filled-relay-ev.entity.ts

```diff
@@ -0,0 +1,55 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, Unique, UpdateDateColumn } from "typeorm";
+
+export type FilledRelayEventArgs = {
```

Comment:
NIT: Can't we get this from the typechain bindings in `@across-protocol/contracts-v2`?

Reply:
Only if we install 2 different versions of the contracts-v2 package because we need to support all the events from all the contracts versions

Reply:
I decided the scraper to have its own contract types since these don't change often